packages/fxa-payments-server/src/components/PaymentConsentCheckbox/index.test.tsx

```diff
@@ -71,23 +71,52 @@ describe('components/PaymentConsentCheckbox', () => {
 
   describe('Legal', () => {
     describe('rendering the legal checkbox Localized component', () => {
-      function runTests(
-        plan: Plan,
-        expectedMsgId: string,
-        expectedMsg: string
-      ) {
+      function runTests(plan: Plan, expectedMsgId: string) {
         const props = { plan };
 
         const testRenderer = TestRenderer.create(<WrapCheckbox {...props} />);
         const testInstance = testRenderer.root;
         const legalCheckbox = testInstance.findByProps({ id: expectedMsgId });
         const expectedAmount = getLocalizedCurrency(plan.amount, plan.currency);
+        const planPricing = formatPlanPricing(
+          plan.amount,
+          plan.currency,
+          plan.interval,
+          plan.interval_count
+        );
 
         expect(legalCheckbox.props.vars.amount).toStrictEqual(expectedAmount);
         expect(legalCheckbox.props.vars.intervalCount).toBe(
           plan.interval_count
         );
-        expect(legalCheckbox.props.children.props.children).toBe(expectedMsg);
+
+        expect(legalCheckbox.props.children.props.children[0]).toBe(
+          'I authorize Mozilla, maker of Firefox products, to charge my payment method '
+        );
+        expect(
+          legalCheckbox.props.children.props.children[1].props.children
+        ).toBe(planPricing);
```

Comment:
Instead of calculating `planPricing` in this function, could you pass it in as an argument instead?

Also, instead of calculating it using `formatPlanPricing`, could you set it as a constant? I quickly checked, and it doesn't look like we have tests for `formatPlanPricing` so if we break something in that function, it won't get picked up anywhere. (Looks like these tests double as a test of that function)